src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/index.md

```diff
@@ -0,0 +1,152 @@
+
+The Kubernetes agent is installed using [Helm](https://helm.sh) via the [octopusdeploy/kubernetes-agent](https://hub.docker.com/r/octopusdeploy/kubernetes-agent) chart.
+
+To simplify this, there is an installation wizard in Octopus to generate the required values.
+
+:::div{.warning}
+Helm will use your current kubectl config, so make sure your kubectl config is pointing to the correct cluster before executing the following helm commands.
+You can see the current kubectl config by executing:
+```
+kubectl config view
+```
+:::
+
+### Configuration
+
+1. Navigate to **Infrastructure ➜ Deployment Targets**, and click **Add Deployment Target**.
+2. Select **KUBERNETES** and click **ADD** on the Kubernetes Agent card.    
+3. This launches the Add New Kubernetes Agent dialog
+
+:::figure
+![Kubernetes Agent Wizard Config Page](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/kubernetes-agent-wizard-config.png)
+:::
+
+1. Enter a unique display name for the target. This name is used to generate the Kubernetes namespace, as well as the Helm release name
+2. Select at least one [environment](/docs/infrastructure/environments) for the target.
+3. Select at least one [target tag](/docs/infrastructure/deployment-targets/#target-roles) for the target.
+4. Optionally, add the name of an existing [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) for the agent to use. The storage class must support the ReadWriteMany [access mode](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).  
+If no storage class name is added, the default Network File System (NFS) storage will be used.
+
+#### Advanced options
+
+:::figure
+![Kubernetes Agent default namespace](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/kubernetes-agent-default-namespace.png)
+:::
+
+You can choose a default Kubernetes namespace that resources are deployed to. This is only used if the step configuration or Kubernetes manifests don’t specify a namespace.
+
+##### Octopus Server (self-hosted)
+
+For self-hosted installations of Octopus Server, the Server URL and Communications URL can be set. See [here](/docs/infrastructure/deployment-targets/tentacle/tentacle-communication#polling-tentacles) for more information.
+
+:::figure
+![Kubernetes Agent self-hosted urls](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/kubernetes-agent-urls.png)
+:::
+
+### NFS CSI driver
+
+If no [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) name is set, the default NFS storage pod will be used. This runs a small NFS pod next to the agent pod and provides shared storage to the agent and script pods.
+
+A requirement of using the NFS pod is the installation of the [NFS CSI Driver](https://github.com/kubernetes-csi/csi-driver-nfs). This can be achieved by executing the presented helm command in a terminal connected to the target Kubernetes cluster.
```

Comment:
It might be worth adding the command here so they can copy + paste it rather than having to go back to and start another wizard.